src/pretix/plugins/stripe/tasks.py

```diff
@@ -54,13 +54,28 @@ def stripe_verify_domain(event, domain):
     prov = StripeCC(event)
     account = get_stripe_account_key(prov)
 
+    # Yes, we could just use the **prov.api_kwargs
+    # But since we absolutely need to always issue this call with live keys,
+    # we're building our api_kwargs here by hand.
+    # Only if no live connect secret key is set, we'll fall back to the testmode keys.
+    # But this should never happen except in scenarios where pretix runs in devmode.
+    if prov.settings.connect_client_id and prov.settings.connect_user_id:
+        api_kwargs = {
+            'api_key': prov.settings.connect_secret_key or prov.settings.connect_test_secret_key,
+            'stripe_account': prov.settings.connect_user_id
+        }
+    else:
+        api_kwargs = {
+            'api_key': prov.settings.secret_key,
```

Comment:
What if we're on a self-hosted system (no Stripe Connect) and the user entered a test secret key? Shouldn't we skip creating the Apple Pay domain entirely in this case?

Reply:
As is, it would still create the domain in testmode and not save to our model - which is fine, since it does enable testing of apple pay... At least that's how I understand it...